app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -398,4 +402,35 @@ def save_node_selector_parts_inventory(entity, hashes, target = nil)
     save_inventory_multi(entity.node_selector_parts, hashes, deletes, [:section, :name])
     store_ids_for_new_records(entity.node_selector_parts, hashes, [:section, :name])
   end
+
+  def save_container_builds_inventory(ems, hashes, target = nil)
+    return if hashes.nil?
+    target = ems if target.nil?
+
+    ems.container_builds(true)
+    deletes = target.kind_of?(ExtManagementSystem) ? ems.container_builds.dup : []
+
+    hashes.each do |h|
+      h[:container_project_id] = h.fetch_path(:project, :id)
+    end
+    save_inventory_multi(ems.container_builds, hashes, deletes, [:ems_ref], [],
+                         [:labels, :project, :resources])
```

Comment:
If you indeed want to save labels for container_builds as you defined in your model classes (has_many :labels)
you should probably pass them as child_keys and not ignore_keys